### PR TITLE
Issue 1922: Fix typo 'Building the Code' Dev Guide

### LINKED ISSUE #1922: Documentation task: Fix typo on 'Building the Code' in the Developer Guide
# Task Description
The 'Building the Code' section in the Developer Guide has a slight typo:

To execute a particular **tests**, run:

Should be:
To execute a particular **test**, run:

# Tasks

The following tasks will need to be carried out:
* [ ] Change 'tests' to 'test'
# Useful Links

* [Building the Code](https://strongbox.github.io/developer-guide/building-the-code.html)

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @sbespalov
  * @steve-todorov

### PR DESCRIPTION
**Pull request by jeffreyperkinson**
Closes strongbox/strongbox#1922.

### CODE REVIEW

**Comment by carlspring** [comment]
Are there perhaps other bits of the documentation that you would like to help polish up?

**Comment by jeffreyperkinson** [comment]
Sure thing! I can make note as I go or work on specific tickets if you’d like when I have some time.

**Comment by carlspring** [comment]
I meant as part of this pull request...

**Comment by jeffreyperkinson** [comment]
Of course.  My mistake.

**Comment by steve-todorov** [comment]
Hey @jperk224 - did you find any more typos or should we proceed with merging? :)

**Comment by jeffreyperkinson** [comment]
Yep I have a couple more locally. I will try to push them up tonight.

**Comment by jeffreyperkinson** [comment]
Hey @steve-todorov I've pushed what I have.  Thank you.

File: docs/developer-guide/ide-setup/eclipse.md [hunk 1]
```diff
@@ -70,4 +70,4 @@ Select `strongbox-masterbuild` project, Right click -> Maven -> Update Project
 [AQL groovy and twig Build Path]: {{assets}}/screenshots/ide/eclipse-setup/15-eclipse-aql-groovy-twig-to-build-path.png "AQL groovy and twig Build Path"
 [AQL groovy libraries in Build Path]: {{assets}}/screenshots/ide/eclipse-setup/16-eclipse-aql-groovy-libraries.png "AQL groovy libraries in Build Path"
 [AQL groovy project nature]: {{assets}}/screenshots/ide/eclipse-setup/17-eclipse-aql-groovy-nature.png "AQL groovy project nature"
-[masterbuild Maven update]: {{assets}}/screenshots/ide/eclipse-setup/18-eclipse-masterbuild-maven-update.png "masterbuild Maven update"
\ No newline at end of file
+[masterbuild Maven update]: {{assets}}/screenshots/ide/eclipse-setup/19-updated-eclipse-masterbuild-maven-update.png "masterbuild Maven update"
```
Comment (carlspring) [inline docs/developer-guide/ide-setup/eclipse.md]: Can't you just use the old link and just overwrite this file, or am I missing something...?

Comment (jeffreyperkinson) [reply docs/developer-guide/ide-setup/eclipse.md]: Yep I can. I was reluctant to delete any existing asset. I can make that change certainly.

**Comment by jeffreyperkinson** [comment]
Reverted screen shot link and added newline at EOF plus one more typo fix.

**Pull request closed without merging**